instrumentation/google.golang.org/grpc/otelgrpc/filters/interceptor/filters.go

```diff
@@ -0,0 +1,167 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
I just moved the original `filters.go` and `filters_test.go` to `go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc/filters/interceptor` because interceptor-based tracing is deprecated.